string class_name
bool has_obstacle       #the detected object has obstacle behind it or not
float64 X_w             #define the world coordinate and body coordinate
float64 Y_w
float64 Z_w
float64 X_b
float64 Y_b
float64 Z_b

